packages/Popover/src/hooks.ts

```diff
@@ -56,3 +56,18 @@ export function useMutationObserver<Value>(
 
   return value;
 }
+
+export function useElementNode<ElementType = HTMLElement>(): [
```

Comment:
I'm wondering what the value of having the person using the hook pass a type provides. What was the thought behind this?

It doesn't appear that we treat any element type differently, so couldn't we omit this?

Reply:
If you wanted to be more exact about the type you have the ability to do so.

Reply:
I seem to have a case of the Fridays today. Completely missed the point of this in that we use `ElementType` to type the returned value. 👍